source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. Python Zeppelin documentation master file, created by
+   sphinx-quickstart on Mon May 22 13:57:54 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Python Zeppelin's documentation!
+===========================================
+
```

Comment:
Could you also add a reference to the zeppelin module here? I think it would be handy for a user to find it 😺